front/lib/api/assistant/agent.ts

```diff
@@ -1205,7 +1200,7 @@ async function* runAction(
   }
 }
 
-export class TokenEmitter {
+export class AgentMessageContentParser {
```

Comment:
Renaming this class, since it's no longer only emitting tokens, but also parsing content.